test/e2e/cmd/run/run.go

```diff
@@ -484,3 +496,39 @@ func (h *helper) dumpK8sData() {
 		log.Error(err, "Failed to run hack/eck-dump.sh")
 	}
 }
+
+func (h *helper) cleanupLogFile() func() {
+	return func() {
+		log.Info("Cleaning up file with test log output")
+		content, err := ioutil.ReadFile(testsLogFile)
```

Comment:
I would use a `bufio.Reader` and call `ReadBytes('\n')` to avoid reading the entire file into memory at once.

Reply:
Fixed